src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -216,9 +216,9 @@
     <PlatformManifestFileEntry Include="icudt_EFIGS.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="icudt_optimal.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="icudt_optimal_no_CJK.dat" IsNative="true" />
-    <PlatformManifestFileEntry Include="binding_support.js" IsNative="true" />
-    <PlatformManifestFileEntry Include="dotnet_support.js" IsNative="true" />
-    <PlatformManifestFileEntry Include="library_mono.js" IsNative="true" />
+    <PlatformManifestFileEntry Include="binding-support.js" IsNative="true" />
+    <PlatformManifestFileEntry Include="dotnet-support.js" IsNative="true" />
+    <PlatformManifestFileEntry Include="library-mono.js" IsNative="true" />
```

Comment:
Why the change from `_` to `-`?

Reply:
@vargaz suggested it since the rest of the repo uses this convention

Reply:
ah, that's correct. I was only looking at the `icudt_*` just above these lines!